lib/core/src/sdk.rs

```diff
@@ -1031,6 +1045,16 @@ impl LiquidSdk {
             compressed: true,
             inner: refund_keypair.public_key(),
         };
+        let webhook = self.persister.get_webhook_url()?.map(|url| ChainWebhook {
+            url,
+            hash_swap_id: Some(true),
+            status: Some(vec![
+                ChainSwapStates::TransactionConfirmed,
```

Comment:
As far as I understand TransactionConfirmed is sent when the user lockup is confirmed. I don't think we need to do anything at this state as we are waiting for the server transaction to confirm. Perhaps we need to omit it too?

Reply:
True, this is our lockup confirmation and is already recorded